src/utils/aspect-ratio-support.tsx

```diff
@@ -26,3 +27,92 @@ export const AspectRatioSupportProvider: React.FC<Props> = ({children}) => {
 };
 
 export const useSupportsAspectRatio = (): boolean => React.useContext(AspectRatioSupport);
+
+const useAspectRatioStyles = createUseStyles(() => ({
+    container: {
+        '@supports (aspect-ratio: 1 / 1)': {
+            aspectRatio: ({aspectRatio}) => aspectRatio ?? 'unset',
```

Comment:
I'm not that sure but maybe you can set `undefined` in case it's not needed, and the property won't be set at all?

Reply:
`unset` is the value we should use here, https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#syntax

and as we are switching from jss to vanilla extract, better keep this with the standard